streaming/base/converters/__init__.py

```diff
@@ -3,6 +3,7 @@
 
 """Utility function for converting spark dataframe to MDS dataset."""
 
-from streaming.base.converters.dataframe_to_mds import MAPPING_SPARK_TO_MDS, dataframeToMDS
+from streaming.base.converters.dataframe_to_mds import (MAPPING_SPARK_TO_MDS, dataframe_to_mds,
+                                                        dataframeToMDS)
 
-__all__ = ['dataframeToMDS', 'MAPPING_SPARK_TO_MDS']
+__all__ = ['dataframeToMDS', 'dataframe_to_mds', 'MAPPING_SPARK_TO_MDS']
```

Comment:
should `'dataframeToMDS'` be in this list? I thought it was being deprecated?

Reply:
Yes, but it is going to be deprecated after 0.8. so will keep it for a while. :)